docs/develop/typescript/failure-detection.mdx

```diff
@@ -1,164 +1,152 @@
 ---
 id: failure-detection
-title: Failure detection
+title: Failure detection - TypeScript SDK feature guide
 sidebar_label: Failure detection
-slug: /develop/typescript/failure-detection
-toc_max_heading_level: 2
+description: Learn how to set Workflow and Activity Timeouts, implement Retry Policies, and manage Activity Heartbeats using the Temporal TypeScript SDK for efficient Workflow Execution.
```

Comment:
One of the things we do on updates is throw descriptions into SEO rewording. Here's what I got back. You can ignore this without any real consequence.

> Optimize Workflow Execution with Temporal TypeScript SDK: Set Timeouts, Retry Policies, and manage Activity Heartbeats efficiently.